app/helpers/routerHelper.js

```diff
@@ -7,6 +7,8 @@ export const withCatch = (authProvider, route) => async (req, res, next) => {
   try {
     return await route(req, res, next);
   } catch (err) {
+    let responseData;
+
```

Comment:
Can we move this declaration a bit closer to its assignment, say before the `if (err.response)`?